### ISSUE TITLE
Enhancement on page /Configuring.html

### ISSUE THREAD

**Issue by Quafadas** [krassowski/jupyterlab-lsp#648]
How to setup "initialise" parameters, in my case for metals. 

According to the metals docs 
https://scalameta.org/metals/docs/integrations/new-editor#initialize

It should be possible exchange some settings on initialization, which dictate the "features" enabled, or triggered in the language kernel, for example 
```
"compilerOptions.completionCommand" : "editor.action.triggerSuggest"
```
How does jupyterlab-lsp negotiate initilisation? Is is possible to sent such commands to the language server on initialisation?

**Comment by krassowski** [comment]
Yes. You can go to _Advanced Settings Editor_ and configure it the settings of the server (which is described in the README, [here](https://github.com/krassowski/jupyterlab-lsp#configuring-the-servers) but missing from the RTD docs indeed). For metals I guess you would use something like:

```json
{
  "language_servers": {
    "metals": {
      "serverSettings": {
        "compilerOptions.completionCommand" : "editor.action.triggerSuggest"
      }
    }
  }
}
```

I see how confusing this can be because when the docs says "servers configuration" it means "making your LSP server get detected properly" but users might be also interested in "configuring the LSP server options" which is not covered there. If you would like to improve the docs please feel free to open a PR.
